en/firefox/browsers/compare/safari.ftl

```diff
@@ -0,0 +1,74 @@
+compare-safari-privacy-has-become-a-white = Privacy has become a white hot topic for tech companies as they realize more and more people are feeling vulnerable to things like data breaches, ad trackers and hackers. But when it comes down to the real tools people use to navigate the actual interwebs, is it all talk or are they actually taking action to keep your data secure?
+compare-safari-as-alluded-to-before-apple-fallback = As alluded to before, { -brand-name-apple } is one of those companies that recently decided to step up their privacy game. Not long ago, { -brand-name-apple } implemented cross-site tracking prevention in { -brand-name-safari }, which prevents ads from following you around the internet. { -brand-name-safari } also offers a strong password suggestion when you sign up for a new account on any website. And if you’re invested in the iCloud ecosystem, it syncs that password securely with your other devices, so you never actually have to remember it.
+
+# "Standing on the soap box" means trying to spread a persuasive message to the masses. Alternative: "We've been spreading the message about privacy for a long time." Soap used to come in wooden crates, and this idiom refers to activists or preachers who would stand on an empty soap box on a street corner to speak to a crowd.
+compare-safari-like-safari-we-at-firefox = Like { -brand-name-safari }, we at { -brand-name-firefox } have made a point of focusing on privacy and security. But unlike { -brand-name-safari }, we’ve been standing on the privacy soap box for a long time. In fact, { -brand-name-mozilla } (our parent company) was one of the first voices in the tech community to sound the alarm for online privacy.
+
+# Variables:
+#   $attrs (string) - link to https://addons.mozilla.org/firefox/addon/facebook-container/ with additional attributes for analytics
+# "WebMD" is a brand name.
+compare-safari-our-private-browsing-mode = Our Private Browsing mode blocks trackers and erases your passwords, cookies and history every time you close it. But you can also experience our advanced privacy features even in regular browsing mode. With the latest edition of { -brand-name-firefox }, enhanced tracking prevention is turned on by default. This prevents things like cross-site trackers from following you as you jump around the web. Also, with { -brand-name-facebook } being caught out almost daily for privacy problems, our <a { $attrs }>{ -brand-name-facebook-container }</a> extension makes a lot of sense. It makes it harder for { -brand-name-facebook } to track you around the web — similar to what Safari does to prevent cross-site tracking — but Firefox actually isolates your { -brand-name-facebook } session into a separate container blocking { -brand-name-facebook } from tracking what you do on other websites. Why do they need to know what you look up on WebMD anyway?
```

Comment:
`Safari`, `Firefox` are missed towards the end of the paragraph.

Reply:
This one is still not fixed:
1). `... similar to what Safari does to prevent cross-site...`
2). ...`but Firefox actually isolates your { -brand-name-facebook } `